src/test/java/org/ibp/api/java/impl/middleware/name/GermplasmNameTypeValidatorTest.java

```diff
@@ -0,0 +1,159 @@
+package org.ibp.api.java.impl.middleware.name;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.generationcp.middleware.api.nametype.GermplasmNameTypeDTO;
+import org.generationcp.middleware.api.nametype.GermplasmNameTypeRequestDTO;
+import org.generationcp.middleware.api.nametype.GermplasmNameTypeService;
+import org.ibp.api.exception.ApiRequestValidationException;
+import org.ibp.api.java.impl.middleware.germplasm.validator.GermplasmImportRequestDtoValidator;
+import org.ibp.api.java.impl.middleware.name.validator.GermplasmNameTypeValidator;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GermplasmNameTypeValidatorTest {
+
+	@Mock
+	private GermplasmNameTypeService germplasmNameTypeService;
+
+	@InjectMocks
+	GermplasmNameTypeValidator germplasmNameTypeValidator;
+
+	@Before
```

Comment:
If I am not mistaken, initMocks is not required. Could you please try it? 
@cuenyad